src/icalendar/tests/events/issue_464_invalid_rdate.ics

```diff
@@ -0,0 +1,7 @@
+BEGIN:VEVENT
+SUMMARY:RDATE period
+DTSTART:19961230T020000Z
+DTEND:19961230T060000Z
+UID:rdate_period
+RDATE;VALUE=PERIOD:19970101T180000Z/19970102T070000Z,199709T180000Z/PT5H30M

```

Comment:
So, the problem we have here is that one element in this list is actually ok and the other one is not. How can that be reflected?

Is it ok to skip the whole RDATE ... probably is for now. It should be rare to have an invalid rdate.